pkg/model/model.go

```diff
@@ -77,7 +77,6 @@ type IstioDependencies struct {
 	ClientGo       *Dependency `json:"client-go"`
 	TestInfra      *Dependency `json:"test-infra"`
 	Tools          *Dependency `json:"tools"`
-	Envoy          *Dependency `json:"envoy"`
```

Comment:
This removes the manifest from the manifest, yaml which contains the Envoy sha today (from the envoyproxy/envoy repo). See https://storage.googleapis.com/istio-prerelease/prerelease/1.19.3/manifest.yaml for an example.